src/main/java/org/zone/ZonePlugin.java

```diff
@@ -208,6 +219,26 @@ public void onRegisterCommands(@SuppressWarnings("BoundedWildcard") final Regist
                 "protect");
     }
 
+    @Listener
+    public void onRegisterData(RegisterDataEvent event) {
+        event.register(DataRegistration.of(ZoneKeys.HUMAN_AI_ATTACHED_ZONE_ID, Human.class));
+    }
+
+    @Listener
+    public void onReload(RefreshGameEvent event) {
+        Optional<Audience> cSender = event.cause().first(Audience.class);
+        try {
+            this.config.getLoader().load();
+            cSender.ifPresent(audience -> audience.sendMessage(Messages.getZoneConfigReloadedInfo()));
+            this.zoneManager.load(this.zoneManager.zonesReload());
```

Comment:
The reload command already registers the zone. No need to load it.

Reply:
K..